src/main/kotlin/solve/settings/visualization/fields/view/VisualizationSettingsLayerCell.kt

```diff
@@ -0,0 +1,240 @@
+package solve.settings.visualization.fields.view
+
+import javafx.application.Platform
+import javafx.geometry.Pos
+import javafx.scene.Node
+import javafx.scene.Parent
+import javafx.scene.Scene
+import javafx.scene.control.Button
+import javafx.scene.control.ComboBoxBase
+import javafx.scene.control.Label
+import javafx.scene.image.Image
+import javafx.scene.paint.Color
+import javafx.scene.text.Font
+import javafx.util.Duration
+import org.controlsfx.control.PopOver
+import solve.scene.controller.SceneController
+import solve.scene.model.LandmarkType
+import solve.scene.model.LayerSettings
+import solve.settings.visualization.popover.PointLayerSettingsPopOverNode
+import solve.utils.*
+import solve.utils.nodes.DragAndDropListCell
+import solve.utils.structures.Point
+import tornadofx.*
+
+class VisualizationSettingsLayerCell(
+    private val sceneController: SceneController
+) : DragAndDropListCell<LayerSettings>() {
+    companion object {
+        private const val LayerFieldHeight = 30.0
+        private const val LayerFieldNameMaxWidth = 80.0
+        private const val LayerFieldNameFontSize = 13.0
+        private const val LayerFieldEditIconSize = 18.0
+        private const val LayerVisibilityIconSize = 22.0
+        private const val LayerIconWidth = 25.0
+
+        private const val LayerFieldHBoxPaddingRight = -2.5
+        private const val LayerTypeIconPaddingRight = 5.0
+        private const val LayerVisibilityIconPaddingLeft = -5.0
+
+        private val LayerSettingsSpawnPositionOffset = Point(-20.0, 30.0)
```

Comment:
When I reimport project, settings modal window appears in a wrong position.
![3LCFOSpqOk](https://user-images.githubusercontent.com/55746901/222921736-4572cc41-1c5c-46b0-9d25-f13ca25ae134.gif)